tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AppTrimmingTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// <copyright file="AppTrimmingTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Linq;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+#if NET7_0
```

Comment:
Trimming is only supported in .NET 7?

Reply:
Nope, my intentions it’s to support from net6.0 and I think I tested for net7.0 because in CI we test for netcoreapp3.1 and net7.0 according to the name of the stages.

Reply:
I think we run either all runtimes (including `net6.0`) or a reduced set depending on what triggered the build (PR vs `master` vs scheduled).

@andrewlock?

Reply:
Yup good call, I will try to do that then

Reply:
done in [c9f0b83](https://github.com/DataDog/dd-trace-dotnet/pull/4195/commits/c9f0b839c2a5105980222727f8de6af6070cabc5)